docs/App Service Provider Guide/index.md

```diff
@@ -0,0 +1,90 @@
+# App Service Provider Guide
+
+### App Service RPCs
+
+There are currently four RPCs related to app services which must be supported by every ASP:
+
+#### PublishAppService
+
+**Direction:** *ASP -> Core*
+
+This function is sent by the ASP to initially create the service. This is where the service's manifest is defined, which includes the type of data provided by the service as well as what RPCs can be handled by the service.
+
+#### GetAppServiceData
+
+**Direction:** *ASC -> Core -> ASP*
+
+This function is sent to the ASP when an ASC sends this message. The ASP is expected to respond to this message with its most recent service data. 
+
+The ASP can receive this message _only_ when its service is active.
+
+#### OnAppServiceData
+
+**Direction:** *ASP -> Core -> ASC*
+
+This notification is expected to be sent by the ASP whenever there are any significant changes to its service data _or_ if its service becomes active. Core will forward this message to any ASCs that have subscribed to data for this service type. 
+
+The ASP is expected to send this message _only_ when its service is active.
+
+#### PerformAppServiceInteraction
+
+**Direction:** *ASC -> Core -> ASP*
+
+This function is sent to the ASP when an ASC sends this message with the ASP's specific service ID. This indicates that the ASC wishes to perform a service-specific function on the ASP. The API for such interactions must be defined by the ASP separately. The ASP is expected to process this message and respond with `SUCCESS` or return an error response if the interaction was not successful. 
+
+This ASP can receive this message regardless of whether its service is active, since it is directed at a specific service.
+
+#### RPC Passing
+
+There are a number of existing RPCs which are allowed to be handled by the ASP based on service type:
```

Comment:
Technically there's no particular restriction on which RPC requests can be handled by an ASP. But if we're encouraging the restrictions set in the spec then I guess this is fine

Reply:
It is restricted by the OEM, and I think that practically they wouldn't allow RPC passthrough for requests beyond what we have already defined.